src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiBuildTypeQuery.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@StructPart
+public class KojiBuildTypeQuery
+        extends KojiQuery
+{
+    @DataKey( "query" )
```

Comment:
Actually, query itself is a map.  But I think whatever else we said about needing a custom handler still stands.

```
<value>
  <struct>
    <member>
      <name>name</name>
      <value>rpm</value>
    </member>
    <member>
      <name>id</name>
      <value>1</value>
    </member>
  </struct>
</value>
```